data_generator.py

```diff
@@ -231,6 +233,20 @@ def __getitem__(self, idx):
                 img, ann['bbox'])
             transformed_label = self.transform_label(
                 ann['keypoints'], cropped_width, cropped_height, anchor_x, anchor_y)
+            if(self.is_eval):
```

Comment:
python code guidelines suggest no parentheses:
```suggestion
            if self.is_eval:
```